Source/Core/Core/PowerPC/SignatureDB/MEGASignatureDB.cpp

```diff
@@ -96,7 +96,6 @@ bool GetRefs(MEGASignature* sig, std::istringstream* iss)
 
     ref_count += 1;
     num.clear();
-    ref.clear();
```

Comment:
This change doesn't really make sense. `ref` is in a valid (though unspecified) state after being moved from, and clearing .clear() on it is fine.

That said, clearing the string appears to be useless as we're assigning a new string to ref in GetFunctionName. So this change isn't entirely nonsensical but the rationale is incorrect.

Reply:
@leoetlino You can close this PR and delete `ref.clear()` yourself, and I will code review for another PR.